labs/lab08-airgap-lite/README.md

```diff
@@ -0,0 +1,608 @@
+
+Next, we'll collect a license file, a download link, and a public kURL bundle.
+
+![airgap-slide-2](img/airgap-slide-2.png)
+
+From there, we'll move all three artifacts into the datacenter via a jump box.
+
+![airgap-slide-3](img/airgap-slide-3.png)
+
+The above diagram shows a three node cluster, but we'll use only a single node.
+While the KOTS bundle will be moved onto the server via SCP as in the diagram,
+the app bundle and license file will be uploaded via a browser UI through an SSH tunnel.
+
+***
+## Getting Started
+
+> **Note:** If you've already completed [Lab 0](../lab00-hello-world), you can skip to [Instance Overview](#instance-overview).
+
+You should have received an invite to log into https://vendor.replicated.com -- you'll want to accept this invite and set your password.
+
+Now, you'll need to install the **Replicated CLI** and set up two environment variables to interact with vendor.replicated.com. See [Get Started -> Steps 1 and 2](https://github.com/replicatedhq/kots-field-labs/blob/main/labs/lab00-hello-world/README.md)
+
+
+`REPLICATED_APP` should be set to the app slug from the Settings page. You should have received your App Name
+ahead of time.
+
+![kots-app-slug](img/application-slug.png)
+
+`REPLICATED_API_TOKEN` should have been provided ahead of time or during the working session.
+
+Once you have the values,
+set them in your environment.
+
+```
+export REPLICATED_APP=...
+export REPLICATED_API_TOKEN=...
+```
+
+Lastly before continuing make sure to clone this repo locally as we will be modifying `lab05` later during the workshop.
+```bash
+git clone https://github.com/replicatedhq/kots-field-labs
+cd kots-field-labs/labs/lab05-airgap
+```
+
+***
+## Instance Overview
+
+You will have received the IP of a jump box and the name of an Air Gap server.
+For example, you may have received:
+
+```text
+dx411-dex-lab05-airgap-jump = 104.155.131.205
+dx411-dex-lab05-airgap
+```
+
+In general, the name of the private server will be the same as the jump box, with the characters `-jump` removed from the suffix.
+Put another way, you could construct both instance names programatically as
+
+```shell
+${REPLICATED_APP}-lab05-airgap-jump
+```
+
+and
+
+```shell
+${REPLICATED_APP}-lab05-airgap
+```
+
+### Connecting
+
+First set your application slug, and the public IP of your jump box:
+
+```shell
+export JUMP_BOX_IP=...
+export REPLICATED_APP=... # your app slug
+```
+
+Next, you can SSH into the Air Gap server using the following command:
+
+```shell
+ssh -J kots@${JUMP_BOX_IP} kots@${REPLICATED_APP}-lab05-airgap
+```
+> **NOTE**: You will be prompted **twice** to change the password on first login. Once for the JUMP_BOX and again for the AIRGAP server.
+
+Once you're on the Air Gap server, you can verify that the server indeed does not have internet access. Once you're convinced, you 
+can ctrl+C the command and proceed to the next section
+
+```shell
+curl -v https://kubernetes.io
+```
```

Comment:
Replace with

First set your application slug, the public IP of your jump box and your first name:

```shell
export JUMP_BOX_IP=...
export REPLICATED_APP=... # your app slug
export FIRST_NAME=... # your first name
```

Next, you can SSH into the Air Gap server using the following command:

```shell
ssh -J ${FIRST_NAME}@${JUMP_BOX_IP} ${FIRST_NAME}@${REPLICATED_APP}-lab08-airgap-lite
```

The `-J` option, allows to connect to the target host by first making a ssh connection to the jump host (`${JUMP_BOX_IP}`) described by destination and then establishing a TCP forwarding to the ultimate destination (`${REPLICATED_APP}-lab08-airgap-lite`) from there.

You can also do it in multiple steps and achieve the same:

```shell
local> ssh ${FIRST_NAME}@${JUMP_BOX_IP}
jump> export REPLICATED_APP=...
jump> ssh ${REPLICATED_APP}-lab08-airgap-lite
```

Once you're on the Air Gap server, you can verify that the server indeed does not have internet access. Once you're convinced, you 
can ctrl+C the command and proceed to the next section

```shell
curl -v https://kubernetes.io
```